client/.travis.yml

```diff
@@ -0,0 +1,23 @@
+---
+language: node_js
+node_js:
+  - "0.12"
```

Comment:
Ah good point. Might have to add some stuff to our Yggdrasil Travis file so it'll be able to build and run all the JavaScript tests.

- Sean Copenhaver

> On Jan 10, 2016, at 4:33 PM, Matthew Philyaw <notifications@github.com> wrote:
> 
> In client/.travis.yml:
> 
> > @@ -0,0 +1,23 @@
> > +---
> > +language: node_js
> > +node_js:
> > +  - "0.12"
> will travis build recursively? If so then may go ahead and change it, but if not may just leave this be until we are building with travis.
> 
> —
> Reply to this email directly or view it on GitHub.
>